docs/openapi/api.yaml

```diff
@@ -41,6 +41,8 @@ tags:
     description: Webhooks for receiving events
   - name: slack
     description: Slack channel management
+  - name: api-keys
+    description: API key management
```

Comment:
This seems to refer (eventually) to a description of a single api-key.  Was this internationally plural (`api-keys` vs `api-key`)?

Reply:
I kept it as api-keys as we could use the same endpoint: /tools/api-keys for both post and get.
`GET /tools/api-keys` retrieves all API keys created by a client.